### ISSUE TITLE
Information about caption in the Java AST

### ISSUE THREAD

**Issue by jmini** [asciidoctor/asciidoctorj#612]
I was looking at the [26.10. Table Caption](http://asciidoctor.org/docs/user-manual/#table-caption) section of the user manual. In particular those two examples:

```
[caption="Table A. "]
.A formal table
|===
|Name of Column 1 |Name of Column 2

|Cell in column 1, row 1
|Cell in column 2, row 1

|Cell in column 1, row 2
|Cell in column 2, row 2
|===
```

```
[caption=]
.A formal table
|===
|Name of Column 1 |Name of Column 2

|Cell in column 1, row 1
|Cell in column 2, row 1

|Cell in column 1, row 2
|Cell in column 2, row 2
|===
```

By looking at the AST (in particular the `org.asciidoctor.ast.Table` instance), I do not see any information about the caption.

I cannot develop in Ruby, but I had a look at [table.rb](https://github.com/asciidoctor/asciidoctor/blob/master/lib/asciidoctor/table.rb). I also have seen commit [e24cafc](https://github.com/asciidoctor/asciidoctor/commit/e24cafc25bffcfc63ec26e98783ed359c4de3df2).

How is the ruby-to-java binding realized?
Isn’t a java method `String getCaption()` missing in the java class corresponding to ` abstract_block.rb`

**Comment by robertpanzer** [comment]
Good catch!
StructuralNode and StructuralNodeImpl will probably just need methods `getCaption()` and and `setCaption(String)` similar to `getTitle()` and `setTitle()`.

**Comment by jmini** [comment]
Thank you a lot for your quick feedback and the pointer to similar code.
=> Pull Request: #613

**Comment by jmini** [comment]
Pull Request #613 is now merged.

What is your policy with this issue?
Should I close it?

Should some Labels and Milestones (something like `1.6.0-alpha.7`) be set?

---
Other question:

Should this change be backported on the `master` branch?

**Comment by robertpanzer** [comment]
The Label is a great idea.

To my own surprise I've just seen that we already have a method `caption()` to get the caption on master, so I think we're fine there.
We have no setter for this, but I think that this is fine, we have no setter for the title either.
And for anything related to extensions I'd like to nudge people towards 1.6.0 earlier than later. :)

**Comment by robertpanzer** [comment]
Fixed with #613. Thanks @jmini 🎉
